cmd/harmony/main.go

```diff
@@ -157,37 +194,85 @@ func raiseFdLimits() error {
 	return nil
 }
 
-func getHarmonyConfig(cmd *cobra.Command) (harmonyConfig, error) {
+func promptForConfigUpgrade() bool {
+	if !terminal.IsTerminal(syscall.Stdin) {
```

Comment:
I think it will be more readable if we can put this if statement in upper function calls. What do you think?